pkg/vdbgen/vdb.go

```diff
@@ -52,23 +52,37 @@ type DBGenerator struct {
 type QueryType string
 
 const (
-	ShardCountKey      QueryType = "shardCount"
-	DBCfgKey           QueryType = "dbCfg"
-	StorageLocationKey QueryType = "storageLocation"
-	SubclusterQueryKey QueryType = "subcluster"
+	ShardCountKey         QueryType = "shardCount"
+	DBCfgKey              QueryType = "dbCfg"
+	StorageLocationKey    QueryType = "storageLocation"
+	NodeCountQueryKey     QueryType = "nodeCount"
+	SubclusterQueryKey    QueryType = "subcluster"
+	KSafetyQueryKey       QueryType = "ksafety"
+	LocalDataSizeQueryKey QueryType = "storageLocationSize"
+	DepotSizeQueryKey     QueryType = "depotSize"
+	CatalogSizeQueryKey   QueryType = "catalogSize"
+	VersionQueryKey       QueryType = "version"
 
 	SecretAPIVersion = "v1"
 	SecretKindName   = "Secret"
 
 	ConfigAPIVersion = "v1"
 	ConfigKindName   = "ConfigMap"
+
+	MaxNodeCountForKSafety0 = 3
 )
 
 var Queries = map[QueryType]string{
 	ShardCountKey:      "SELECT COUNT(*) FROM SHARDS WHERE SHARD_TYPE != 'Replica'",
 	DBCfgKey:           "SHOW DATABASE DEFAULT ALL",
 	StorageLocationKey: "SELECT NODE_NAME, LOCATION_PATH FROM STORAGE_LOCATIONS WHERE LOCATION_USAGE = ?",
+	NodeCountQueryKey:  "SELECT COUNT(NODE_NAME) FROM SUBCLUSTERS",
```

Comment:
Suggest querying from nodes. It will be the same result, just more intuitive.
```suggestion
	NodeCountQueryKey:  "SELECT COUNT(*) FROM NODES",
```